src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/service/BulkPricesServiceTest.kt

```diff
@@ -30,13 +35,26 @@ internal class BulkPricesServiceTest {
 
     whenever(priceRepository.findBySupplierAndEffectiveYear(Supplier.SERCO, 2020)).thenReturn(prices)
 
-    val service = BulkPricesService(priceRepository, timeSourceEffectiveYear2020)
+    val service = BulkPricesService(priceRepository, timeSourceEffectiveYear2020, auditService)
+    ReflectionTestUtils.setField(service, "auditService", auditService)
```

Comment:
We shouldn't need this **ReflectionTestUtils**.  Probably easier to talk about it.

Reply:
Oops, looks like I missed one :D